doc/changes/latest.inc

```diff
@@ -24,6 +24,8 @@ Current (1.0.dev0)
 Enhancements
 ~~~~~~~~~~~~
 
+- Add use_orig_time option to :meth:`mne.Annotations.crop`. (:gh:`10396` by `Michiru Kaneda`_)
```

Comment:
@rcmdnk can you keep on the top of the section the entries from new contributors? or if it's the case of this release can you add use the tag :newcontrib: as for the item below?

besides LGTM

Reply:
OK, I move it to the end of the section.